experiment/find-merge-conflicts.sh

```diff
@@ -65,23 +65,23 @@ while true; do
   done
 done
 
-if [[ -z "${conflicting_prs[@]:-}" ]]; then
+if [[ -z "${conflicting_prs[*]}" ]]; then
```

Comment:
Is this technically a regression on unset arrays?

Reply:
It's set to empty to start with.